packages/cascara/src/ui/ChatMessages/Chat.doc.mdx

```diff
@@ -16,7 +16,7 @@ This is an array of objects used to tell Chat what to display. The `type` prop m
 ---
 title: Chat Messages
 ---
-<Chat
+<ChatOld
```

Comment:
Still says ChatOld

@lordpixel I am encountering a lot of these in code that still say ChatOld. Instead of me going line by line to find all of these, can we please do a search to find all of them and resolve first? Just doing a search in the DOM on this page I see 18 instances.

Reply:
done, thanks for the catch!